packages/react/src/themes/teams/components/Label/labelStyles.ts

```diff
@@ -4,17 +4,20 @@ import { pxToRem } from '../../../../lib'
 import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
 import { LabelProps } from '../../../../components/Label/Label'
 import { LabelVariables } from './labelVariables'
+import { getColorScheme } from '../../colors'
 
 const labelStyles: ComponentSlotStylesInput<LabelProps, LabelVariables> = {
-  root: ({ props: p, variables: v, colors }): ICSSInJSStyle => {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const colors = getColorScheme(v.colorScheme, p.color)
+
     return {
       display: 'inline-flex',
       alignItems: 'center',
       overflow: 'hidden',
       height: v.height,
       lineHeight: v.height,
-      color: colors.foreground,
-      backgroundColor: colors.background,
+      color: colors.backgroundDefault,
+      backgroundColor: colors.foregroundDefault,
```

Comment:
Previously the color scheme constructed for Labels was conditional based on whether or not the color was considered dark or light. It looks as if that code was removed, so I'm curious to see how the Label examples look with the new color scheme mapping. Do you have a screenshot?

Reply:
I see now that Teams grafts some lightColorOverrides for certain natural colors. It still would be good to see what the end result looks like (if you have a screenshot)

Reply:
Hey, here is a screenshot of the Label's color example: 
![image](https://user-images.githubusercontent.com/4512430/56404560-a6bbf480-6267-11e9-9ecc-6f2e88e09933.png)
The important thing is that, for each color we are defining the scheme, by defining the foreground + background default colors. As we are the one defining the scheme, these colors can be properly created, with taking into account the contrast betwone the two. We are still using in some cases the white as a foreground, and in others the black color, but still the scheme is not defined, and currently we aim to have correct values for the things which we are actually using (and no regressions are introduced on those example - default, black and white colors). Btw maybe the orange and green should have white foreground, but at this moment, we don't care much about that so it's open of changes that potentially will be introduced by design

Reply:
I wonder if orange and yellow should be considered "light" any more. The 600 variant for each is actually pretty dark. But, like you said, we are in control of the color scheme mapping so we can adjust it however necessary.